File: src/main/java/io/jsql/orientserver/handler/data_define/CreateFunction.java
package io.jsql.orientserver.handler.data_define;

import io.jsql.orientserver.OConnection;

/**
 * Created by 长宏 on 2017/3/18 0018.
 */
public class CreateFunction {
    public static boolean isme(String sql) {
        String sqll = sql.toUpperCase().trim();
        String list[] = sqll.split("\\s+");
        if (list.length > 2 && list[0].equals("CREATE") && list[1].equals("FUNCTION")) {
            return true;
        }
        return false;
    }
    public static void handle(String sql, OConnection c) {
        c.writeok();
    }
}

File: src/main/java/io/jsql/sql/handler/data_define/CreateFunction.kt
package io.jsql.sql.handler.data_define

import io.jsql.sql.OConnection

/**
 * Created by 长宏 on 2017/3/18 0018.
 */
object CreateFunction {
    fun isme(sql: String): Boolean {
        val sqll = sql.toUpperCase().trim { it <= ' ' }
        val list = sqll.split("\\s+".toRegex()).dropLastWhile { it.isEmpty() }.toTypedArray()
        return list.size > 2 && list[0] == "CREATE" && list[1] == "FUNCTION"
    }

    fun handle(sql: String, c: OConnection) {
        c.writeok()
    }
}
